src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,323 @@
+    factory = in-memory-cache
+    config {
+      cache {
+        maximumSize = 1000
+        ttl = 5000
+      }
+      cacheKey = "product-{param.id}"
+      payloadKey = fetch-product
+    }
+    doAction = fetch-product
+  }
+
+  # Target API
+  fetch-product {
+    # https://github.com/Knotx/knotx-data-bridge/blob/2.0.0/http/action/src/main/java/io/knotx/databridge/http/action/HttpActionFactory.java
+    factory = http
+    config {
+      endpointOptions {
+        path = /product/id
+        domain = webapi
+        port = 8080
+        allowedRequestHeaders = ["Content-Type"]
+      }
+    }
+
+  }
+
+  product-to-body {
+    # https://github.com/Knotx/knotx-fragments/blob/2.0.0/handler/core/src/main/java/io/knotx/fragments/handler/action/PayloadToBodyActionFactory.java
+    factory = payload-to-body
+    config {
+      key = fetch-product
+    }
+  }
+}
+```
+As you can see above, we declare the `get-product-task` task that points to the `fetch-product-with-cache` action node. The action declares one success transition (outgoing graph edge) that points to the `product-to-body` action node.
+The action it performs is `fetch-product-with-cache` which we'll create soon. Then we have `onTransitions`, in our case let's 
+define only `_success` - the action to be performed is `product-to-body`  which we'll also define.
+
+Below the task definition, you can see all necessary action's definitions.
+The `fetch-product` action is responsible for the target RESTful HTTP API invocation. In our case, it is the `http://webapi:8080/product/id`. The action logic is delivered via [HTTP Action](https://github.com/Knotx/knotx-data-bridge/tree/2.0.0/http#http-action) implementation so it is a matter of configuration. 
+Now it is time to add a cache. For this purpose, we define a separate 
+`fetch-product-with-cache`  action that:
+- uses [In-memory Cache Action](https://github.com/Knotx/knotx-fragments/tree/2.0.0/handler#in-memory-cache-action) implementation
+- decorates/wraps the `fetch-product` action (see the `doAction` attribute), it is responsible for caching only
+- declares the key name in the fragment's payload that would contain cached values (it is the decorated action name)
+- configures cache parameters such as maximum size and time to live for stored entries
+cache key
+This action is declared as the task root node. Knot.x defines those kinds of actions as behaviours. They wrap a target action and add some functionality. [Circuit Breaker Action](https://github.com/Knotx/knotx-fragments/tree/2.0.0/handler#circuit-breaker-action) is also the example of default behaviours implementations.
+Finally, we define the `product-to-body` action to rewrite the previously fetched data from the fragment's payload to the response body.
+
+## Why do we need caching?
+We've just declared caching behaviour to the API invocation but you may wonder what are the benefits? Why we're using cache? The general purpose of
```

Comment:
I am wondering if those benefits should not be moved to the beginning of this tutorial.

Reply:
done in https://github.com/Knotx/knotx-website/pull/73